generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts

```diff
@@ -1,14 +1,18 @@
 export function AlertService () {
     this.toast = false;
     /*jshint validthis: true */
-    this.$get = getService;
+    var service = {  getService: getService }
```

Comment:
probably 

`var service = {  getService: getService }
    this.$get = service;
    this.showAsToast = function(isToast) {
        this.toast = isToast;
    };
    getService.$inject = ['$timeout', '$sce', '$translate'];
    return service;`

we can write this way but why this.$get? is that advisable?